base/ltcounts.dtx

```diff
@@ -246,7 +249,19 @@
 %    \end{macrocode}
 % \end{macro}
 %
-% \begin{macro}{\@newctr}
+% \begin{macro}{\c@*}
+% \changes{v1.2a}{2025/01/22}{\cs{c@*} added gh/1632}
+% Make the current counter available as a \LaTeX\ counter
+% with name |*|, so `\alph*` returns the current list
```

Comment:
why *list* counter?

Reply:
because it started out like that :-) but it is now broader, so should be changed.